android/filament-android/src/main/java/com/google/android/filament/Texture.java

```diff
@@ -369,6 +375,15 @@ public void finalize() {
 
     public static final int BASE_LEVEL = 0;
 
+    public class Usage {
```

Comment:
It should be a public *static* class. Public constants/inner classes should be declared at the top of the class.

Reply:
Done.